LockboxXCUITests/SnapshotHelper.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  SnapshotHelper.swift
+//  Example
+//
+//  Created by Felix Krause on 10/8/15.
+//  Copyright © 2015 Felix Krause. All rights reserved.
```

Comment:
>it's a little concerning to add third-party code to our repo without clear licensing

Good catch. And if not spotted already, I believe this is a _little_ misleading as its technically distributed via fastlane/fastlane which is MIT.

https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md#license

Perhaps we can open an issue over there for `@KrauseFx` to help clarify and revise the license language in the helper itself, if appropriate? Otherwise point to that fact somehow here _and_ in our docs, as suggested?

Reply:
> Perhaps we can open an issue over there for @KrauseFx to help clarify and revise the license language in the helper itself, if appropriate?

Yes, that would be good.  I was trying to trace it some, but it was going a little deeper than I was initially prepared for.

> Otherwise point to that fact somehow here and in our docs, as suggested?

I think pointing it out would be worthwhile regardless; a very quick glance at fastlane's site implies any changes here could make upgrading slightly more dififcult (??), which points to adding a note in one of our other doc pages.

Reply:
😬 the worst part of this is that they tell you to include this file in the test target as part of setting up the screenshots? It looks like fxiOS just includes it as-is: https://github.com/mozilla-mobile/firefox-ios/blob/master/fastlane/SnapshotHelper.swift

Reply:
😕

In previous places there's been disputes, so I'd personally rather we take a step or two to acknowledge the "from outside" code.

Filing an issue with Fastlane requires a bit more than I'm prepared to supply right now; @devinreams or @sashei could you do that sometime?

Reply:
Wow, nice catch. We definitely don't want to cause any legal issues with this file. I'm not a lawyer, but according to the LICENSE file of fastlane (https://github.com/fastlane/fastlane/blob/master/LICENSE)

> Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"),

where as `copy of this software` refers to the source code contained in the repo, which includes the `.m` file you mention here

Reply:
Heyy 👋 We just removed copyrights from `SnapshotHelper.swift` (and a bunch of others) in https://github.com/fastlane/fastlane/pull/12795 😊 

Lemme know if there is any other changes you may need! 🚀

Reply:
I'm clearing my concerns here, it's addressed (or will be very soon), so I think we're GTG.

Reply:
Glad to hear, thanks for letting us know 👍